flake.nix

```diff
@@ -0,0 +1,95 @@
+{
+  description = "Traewelling";
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    devenv.url = "github:cachix/devenv";
+  };
+  outputs = inputs @ {flake-parts, ...}:
+    flake-parts.lib.mkFlake {inherit inputs;} {
+      imports = [
+        inputs.devenv.flakeModule
+      ];
+      systems = [
+        "x86_64-linux"
+        "aarch64-linux"
+        "x86_64-darwin"
+        "aarch64-darwin"
+      ];
+      perSystem = {
+        config,
+        pkgs,
+        ...
+      }: {
+        devenv.shells.default = {
+          languages = {
+            php.enable = true;
+            javascript.enable = true;
+          };
+          dotenv.disableHint = true;
+          services.mysql = {
+            enable = true;
+            ensureUsers = [
+              {
+                name = "homestead";
+                password = "secret";
```

Comment:
This only works with the mysql information from `.env.example`. Can this please use the real environment variables? I'm not a flake/devenv expert, so I'm not sure how to handle that, but I'm sure its not a large task for you c: